app/views/games/show.html.erb

```diff
@@ -5,24 +5,26 @@
   </div>
 
     <div class="board-container">
-        <table id="board" data-game_id="<%= @game.id %>">
-        <% 8.times.each do |row| %>
-          <tr id="row">
-            <% 8.times.each do |col| %>
-              <% piece = @game.pieces.where(position_x: col, position_y: row).first %>
-              <% if piece.present? %>
-                <td data-piece_id="<%= piece %>" data-piece_name="<%= piece.color %>_<%= piece.type %>">
-                  <% image ="#{piece.color}_#{piece.type}.png" %>
-                  <%= image_tag(image, class: "piece_image") %>
-                </td>
-              <% else %>
-                <td></td>
+      <table id="board" data-game_id="<%= @game.id %>">
+          <% 8.times.each do |row| %>
+            <tr id="row">
+              <% 8.times.each do |col| %>
+                <% piece = @game.pieces.where(position_x: col, position_y: row).first %>
+                <% if piece.present? %>
+                  <td data-piece_id="<%= piece %>" data-piece_name="<%= piece.color %>_<%= piece.type %>">
+                    <% image ="#{piece.color}_#{piece.type}.png" %>
+                    <%= image_tag(image, class: "piece_image") %>
+                  </td>
+                <% else %>
+                  <td></td>
+                <% end %>
               <% end %>
-            <% end %>
-          </tr>
-        <% end %>
+            </tr>
+          <% end %>
       </table>
     </div>
-  </div>
-</div>
+    
+    
 
+  </div>
+</div>
```

Comment:
This is also different to what I came up with so we should collaborate sometime this week to combine what we have